### ISSUE TITLE
Red progress bar as an option

### ISSUE THREAD

**Issue by acwolff** [fancyapps/fancybox#2126]
For most albums I like the new red progress bar in a slide show very much, but for embedded albums [like this album](https://www.andrewolff.nl/TestAlbum/EmbeddedAlbums/FancyBox/) it is less attractive.

So my request is:

Please make the progress bar an option.
And may be it is also a good idea to make the color selectable.

Please tell me how I can change the color of the progress bar with css, may be I can select a transparant color to hide the progress bar.

**Comment by acwolff** [comment]
Sorry pushen the wrong button

**Comment by Patta** [comment]
@acwolff 
`.fancybox-progress {
  background-color: grey;
}`

**Comment by acwolff** [comment]
Yes that works, see [this embedded albums](https://www.andrewolff.nl/TestAlbum/EmbeddedAlbums/FancyBox/), Thanks!
